packages/cli/__tests__/zostso/__system__/stop/__scripts__/address-space/as_error_stop.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 set -e
 
-zowe zos-tso stop address-space ZOSMFAD-55-aaakaaac
+zowe zos-tso stop address-space --user FakeUser --pass FakePassword --host FakeHost --port 3000
```

Comment:
Probably a dumb question 😢 Where did this change come from? Don't see it in either master or next branch 🤔

Reply:
no idea 🤷‍♂️ ... reverted to be safe